config/webpack.test.js

```diff
@@ -50,7 +50,7 @@ module.exports = {
           options: { esModules: true }
         },
         enforce: 'post',
-        include: path.resolve(process.cwd(), 'src')
+        exclude: [/\.test\.ts$/, /test(ing)?\//]
```

Comment:
presumably this would also exclude a directory called "nottesting" :stuck_out_tongue: 

would a \b "word boundary" work?

/\btest(ing)?\//

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Assertions

Reply:
> "nottesting"

Nice footgun. :smile: 

Thanks though, I didn't know `\b`. Changed.